Error while building docs

Originally submited by @cryptox31

* Commit SHA: ???
* Python version: ???
* Operating System: macOS

### Description

Makefile crashes when building the documentation

### What I Did

```
make install-develop
make docs
```

### What I was expecting to get done
Build the docs

### Additional context
```
➜  task-geo git:(cryptox31/fix/makefile) ✗ make docs
rm -f docs/api/*.rst
/Library/Developer/CommandLineTools/usr/bin/make -C docs clean 2>/dev/null  # this fails if sphinx is not yet installed
sphinx-apidoc --separate --no-toc -o docs/api/ task_geo
Creating file docs/api/task_geo.rst.
Creating file docs/api/task_geo.data_sources.rst.
Creating file docs/api/task_geo.data_sources.noaa.rst.
Creating file docs/api/task_geo.data_sources.noaa.api_connector.rst.
Creating file docs/api/task_geo.data_sources.noaa.api_formatter.rst.
Creating file docs/api/task_geo.data_sources.noaa.ftp_connector.rst.
Creating file docs/api/task_geo.data_sources.noaa.references.rst.
/Library/Developer/CommandLineTools/usr/bin/make -C docs html
Running Sphinx v2.2.0

Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.7/site-packages/sphinx/config.py", line 361, in eval_config_file
    execfile_(filename, namespace)
  File "/opt/anaconda3/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 81, in execfile_
    exec(code, _globals)
  File "/Users/../CoronaWhy/task-geo/docs/conf.py", line 23, in <module>
    import task_geo
ModuleNotFoundError: No module named 'task_geo'

make[1]: *** [html] Error 2
make: *** [docs] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -102,7 +102,7 @@ clean-build: ## remove build artifacts
 	rm -fr dist/
 	rm -fr .eggs/
 	find . -name '*.egg-info' -exec rm -fr {} +
-	find . -name '*.egg' -exec rm -f {} +
+	find . -name '*.egg' -exec rm -fr {} +
 
 .PHONY: clean-pyc
 clean-pyc: ## remove Python file artifacts

